### DISCUSSION TITLE
[Feature Request] Sync with Music app

### DISCUSSION THREAD

**Discussion by citizenserious** [thrillfall/nextcloud-gpodder#70] [Ideas]
The music app has a podcast section, which uses RSS feeds.
So it would be nice if it could get synced (:

Mentioned it here: https://github.com/owncloud/music/issues/975

**Comment by memen45** [comment]
This would be great. Integration would immediately enable access from the other APIs owncloud/music provides (Ampache and Subsonic) as well! 

@thrillfall what would be your considerations towards this idea? 

Would it be possible to merge the API you wrote into the owncloud/music app? Or are there features that conflict with such an approach?
What could be other ways of enabling a sync? Maybe by saving to a file, database or do you have other ideas?

**Reply by thrillfall** [reply]
Nope, you can implement the api also in any other app. Nothing special here.
